client/src/get_auction_info.rs

```diff
@@ -32,13 +32,14 @@ impl<'a, 'b> ClientCommand<'a, 'b> for GetAuctionInfo {
     fn run(matches: &ArgMatches<'_>) {
         let maybe_rpc_id = common::rpc_id::get(matches);
         let node_address = common::node_address::get(matches);
-        let verbose = common::verbose::get(matches);
+        let mut verbosity_level = common::verbose::get(matches);
 
-        let response = casper_client::get_auction_info(maybe_rpc_id, node_address, verbose)
+        let response = casper_client::get_auction_info(maybe_rpc_id, node_address, verbosity_level)
             .unwrap_or_else(|error| panic!("response error: {}", error));
-        println!(
-            "{}",
-            serde_json::to_string_pretty(&response).expect("should encode to JSON")
-        );
+
+        if verbosity_level == 0 {
+            verbosity_level += 1
+        }
```

Comment:
Here, and all other places, why do you increment the verbosity level?

Reply:
The client prints the response regardless of what verbosity level the user requests - it's pretty much all it ever does with the responses.  So the verbosity level of `0` lets the user specify not to print the request, but we need to set it to `1` to get the response printed.

There might be a neater way to do this - I didn't like mutating a user-set config value either, but I didn't want to make it overly complex.